redisinsight/ui/src/components/new-index/CreateIndexStep/CreateIndexStepWrapper.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { fireEvent, render, screen } from 'uiSrc/utils/test-utils'
+import { CreateIndexStepWrapper } from './CreateIndexStepWrapper'
+
+const initSUT = () => render(<CreateIndexStepWrapper />)
+
+describe('CreateIndexStepWrapper', () => {
+  it('should render', () => {
+    const { container } = initSUT()
+
+    expect(container).toBeTruthy()
+
+    // Check if the tabs are rendered
+    const buildNewIndexTabTrigger = screen.getByText('Build new index')
+    const usePresetIndexTabTrigger = screen.getByText('Use preset index')
+
+    expect(buildNewIndexTabTrigger).toBeInTheDocument()
+    expect(usePresetIndexTabTrigger).toBeInTheDocument()
+
+    // Check if the "Use preset index" tab content is selected by default
+    const usePresetIIndexTabContent = screen.queryByText(
+      'TODO: Add content later',
+    )
+    expect(usePresetIIndexTabContent).toBeInTheDocument()
```

Comment:
nit: is there a way to check for some html attribute, as this test would require change when the text changes

Reply:
Sure thing! I've added a `data-testid` to help reliably check for the presence of this element.

Initially, I was using a text-based query, following the default [query rules priority](https://testing-library.com/docs/queries/about/#priority) in React Testing Library, knowing that we'd replace it in a future PR once this component is updated with real content. But you're absolutely right - we'll test the actual contents later when we introduce it, most probably in a separate component.

For now, since we're just verifying that the Tabs component works as expected, using a testid makes perfect sense, and is enough to serve the job. Change applied, thanks for the input! 🙌